test/plugin_functional/test_suites/usage_collection/usage_counters.ts

```diff
@@ -18,7 +18,7 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
 
   async function getSavedObjectCounters() {
     // wait until ES indexes the counter SavedObject;
-    await new Promise((res) => setTimeout(res, 7 * 1000));
+    await new Promise((res) => setTimeout(res, 10 * 1000));
```

Comment:
This test was failing intermittently. Increasing the timeout allows extra time for the types' checks in the SO HTTP APIS.

Reply:
```
const DELAY_IN_SEC = 10; // in case of flakiness, increment this and move on 
```

🙈  :trollface:

Reply:
yea, I know